aws/resource_aws_ssm_document.go

```diff
@@ -39,6 +39,32 @@ func resourceAwsSsmDocument() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validateAwsSSMName,
 			},
+			"attachments": {
```

Comment:
Per comment in read function, recommend naming this `attachments_source` (singular for configuration blocks)

```suggestion
			"attachments_source": {
```